core/src/main/java/com/lafaspot/sparzr/DefaultListener.java

```diff
@@ -0,0 +1,127 @@
+package com.lafaspot.sparzr;
+
+import org.slf4j.Logger;
```

Comment:
Remove all logger call and remove logger from the xml file and from the code, use exceptions or some int/long field that store state if needed for debugging.
If you need to persist some data for debugging, use some state in the classes, but not like a logger more like stats.
You can also use logfast but I don't recommend this for this case.